src/pages/CertificatesViewAll.js

```diff
@@ -5,12 +5,20 @@ import { Grid, Spinner, Table } from '@digicatapult/ui-component-library'
 import Nav from './components/Nav'
 import Header from './components/Header'
 
+import BgMoleculesImageSVG from '../assets/images/molecules-bg-repeat.svg'
 import { Context } from '../utils/Context'
 import { stateToStatus, NameCell } from './components/shared'
 import { personas } from '../App'
 import { useNavigate } from 'react-router-dom'
 import useAxios from '../hooks/use-axios'
-import { formatTimelineDate } from '../utils/helpers'
+import { formatDate, formatCertName, checkCO2Status } from '../utils/helpers'
+
+// TODO call identity?
+const mapOwner = {
+  '5GrwvaEF5zXb26Fz9rcQpDWS57CtERHpNehXCPcNoHGKutQY': 'Heidi',
+  '5FLSigC9HGRKVhB9FiEo4Y3koPsNmBmLJbpXg2mp1hXcS59Y': 'Regulator',
+  '': 'Emma',
+}
```

Comment:
In my case, because I am using the `sqnc-hyproof-api/scripts/1_load_identities.sh` i don't see the need for this. Are you planning to perhaps remove this?

Reply:
removed a comment and this is no longer in question